indexer/db/solana_block_db.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, db: DbConnection):
         self._finalize_query = DbQueryBody()
         self._deactivate_query = DbQueryBody()
         self._activate_query = DbQueryBody()
+        self._priority_fee_percentiles_query = DbQueryBody()
```

Comment:
Please rename to:
`self._select_cu_price_percentile_list_query`

Reply:
done